src/main/scala/li/cil/oc/integration/vanilla/ConverterFluidContainerItem.scala

```diff
@@ -0,0 +1,37 @@
+package li.cil.oc.integration.vanilla
+
+import java.util
+
+import li.cil.oc.{Settings, api}
+import net.minecraft.item
+import net.minecraft.item.ItemStack
+import net.minecraftforge.fluids
+import net.minecraftforge.fluids.FluidStack
+
+import scala.collection.convert.WrapAsScala._
+
+object ConverterFluidContainerItem extends api.driver.Converter  {
+  override def convert(value: AnyRef, output: util.Map[AnyRef, AnyRef]): Unit =
+    value match {
+      case stack: item.ItemStack => stack.getItem match {
+        case fc: fluids.IFluidContainerItem =>
+          output += "capacity" -> Int.box(fc.getCapacity(stack))
+          val fluidStack  = fc.getFluid(stack)
+          if (fluidStack != null && fluidStack.getFluid != null) {
+            val fluid = fluidStack.getFluid
+            if (Settings.get.insertIdsInConverters)
+              output += "fluid_id" -> Int.box(fluid.getID)
+            output += "amount" -> Int.box(fluidStack.amount)
+            output += "fluid_hasTag" -> Boolean.box(fluidStack.tag != null)
+            if (fluid != null) {
+              output += "fluid_name" -> fluid.getName
+              output += "fluid_label" -> fluid.getLocalizedName(fluidStack)
+            }
+          }
+          else
+            output += "amount" -> Int.box(0)
+        case _ =>
+      }
+      case _ =>
+    }
```

Comment:
what do you think about splitting this method to smaller ones? Like extracting branches to their own methods.

Reply:
It does one logical thing and is <20 lines of code, why would I split it?

Reply:
like, I'm open to suggestions, what part would you extract?